tools/ci_build/github/azure-pipelines/templates/win-cpu-ci.yml

```diff
@@ -196,26 +197,32 @@ jobs:
            **/*.pdb
            **/*.dll
 
-      - task: PythonScript@0
-        displayName: 'Generate cmake config'
+      #Manually set msBuildCommandline so that we can also set CAExcludePath
+      #build_dir must be a sub folder of $(Build.SourcesDirectory)
```

Comment:
could you explain why? (perhaps also in the comment)

Reply:
It's an internal thing. I can talk to you offline.